src/components/Icon/Icon.tsx

```diff
@@ -51,22 +52,40 @@ class Icon extends UIComponent<any, any> {
     'circular',
     'className',
     'disabled',
-    'kind',
+    'font',
     'name',
     'size',
     'xSpacing',
   ]
 
   static defaultProps = {
-    as: 'i',
-    kind: 'FontAwesome',
+    as: 'span',
+    size: 'normal',
   }
 
   static rules = iconRules
 
-  renderComponent({ ElementType, classes, rest }) {
+  renderFontIcon(ElementType, classes, rest): React.ReactNode {
     return <ElementType className={classes.root} {...rest} />
   }
+
+  renderSvgIcon(ElementType, classes, rest): React.ReactNode {
+    const icon = svgIcons[this.props.name]
+
+    return (
+      <ElementType className={classes.root} {...rest}>
+        <svg className={classes.svg} viewBox={icon && icon.viewBox}>
```

Comment:
should we allow optional accessibility props here?

Reply:
yes, definitely - this will go next as part of dedicated PR